src/controllers/despesasController.js

```diff
@@ -0,0 +1,34 @@
+const Despesa = require('../model/Despesas.js');
+
+async function cadastrarDespesa(despesa) {
+  return await Despesa.create(despesa);
+}
+async function listarTodos() {
+  try {
+    return await Despesa.find();
+    
+  } catch (error) {
+    return error
+  }
+}
+const despesasControllers = {
+  creat: async (req, res) => {
```

Comment:
Apenas erro de digitação

```suggestion
  create: async (req, res) => {
```